recipes/protobuf/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.22.2":
```

Comment:
I think this should be 4.22.2 for C++.  Protobuf versioning is a bit ad-hoc.  Each language (in this case C++) has its own major version:

https://protobuf.dev/news/2022-05-06/

The v22.x series is a breaking change for C++:

https://protobuf.dev/news/2022-08-03/
https://github.com/protocolbuffers/protobuf/releases/tag/v22.0

The v22.x series is 4.22.x for C++.  The protobuf creates tags (at the same SHA) with the right version for each language:

https://github.com/protocolbuffers/protobuf/releases/tag/v22.2
https://github.com/protocolbuffers/protobuf/releases/tag/v4.22.2
https://github.com/protocolbuffers/protobuf/releases/tag/v3.22.2